bricksrc/relationships.py

```diff
@@ -5,6 +5,10 @@
 Defining Brick relationships
 """
 relationships = {
+    "connectedTo": {
```

Comment:
@gtfierro Is it necessary to specify that 'connectedTo' is a superclass to 'feeds' and 'isFedBy'?

Something like:
```
rdfs:subPropertyOf :feeds, :isFedBy ;
```

Reply:
In my intended design, `feeds` and `isFedBy` would both be subproperties of `connectedTo`, thinking about it more. I'm not sure this is what we actually want to do. If we were to follow this design, then any use of the `feeds` relationship would also imply the existence of a `connectedTo` relationship.

This raises the essential question, which is "does `connectedTo` communicate a bidirectional relationship, or a relationship where the directionality is not known? If its the former, then `connectedTo` should be a sibling of `feeds` and `isFedBy` because it is altogether different. If its the latter, then `connectedTo` should be a superproperty of `feeds` and `isFedBy`

Reply:
We probably want both:
- "feedsAndFedBy" (**we need a better name**) for bidirectional. We want this to work for equipment as well as for spaces/locations
- "connectedTo" for a super property of feeds, isFedby, and feedsAndFedBy that means "we don't know the direction"